editor/src/components/canvas/controls/insert-mode-control-container.tsx

```diff
@@ -249,6 +252,11 @@ export class InsertModeControlContainer extends React.Component<
     )
     const localFrame = Utils.getLocalRectangleInNewParentContext(parentOrigin, dragFrame)
 
+    if (localFrame.width === 0 && localFrame.height === 0) {
+      localFrame.width = DefaultWidth
+      localFrame.height = DefaultHeight
+    }
```

Comment:
Presumably in this case we're inserting with the top-left matching the click point. If that's the case, we should probably centre the inserted element on the click here.

Reply:
Good point, fixed in a8d97c735ed175d05d861d8c2035cec0daceae60 and 3983901e60b13544f97ac334ee6fdf4a002f22f1